src/app/components/popover/index.html

```diff
@@ -92,7 +92,17 @@
   </sky-demo-page-example>
 
 	<sky-demo-page-content sectionHeading="UX guidelines">
-    <p>
+    <div class="sky-subsection-heading">Usage</div>
+		<p>Popovers are appropriate when:</p>
+		<ul>
+			<li>The trigger is client-side, a direct result of a user interaction with an element (such as a click) that does not lead to a change in system state.</li>
```

Comment:
While I used "client-side" and "server-side" as a powerful distinction when I first wrote these guidelines, I imagine that it was more useful to me than it might be to others. Abandoning that language in favor of talking about "changing system state" might be a good direction.